en/docs/api-gateway/1.0.0/setup/storage-and-backends.md

```
@@ -1,60 +1,36 @@
   - configuration
   - devops
 author: WSO2 API Platform Documentation Team
-last_updated: 2026-06-26
+last_updated: 2026-06-11
 content_type: "how-to"
 ---
 
 # Configuring External Storage and Backends
 
-### External Database (Gateway Controller)
+### PostgreSQL (Gateway Controller)
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)